metadata-ingestion/tests/test_helpers/mce_helpers.py

```diff
@@ -55,6 +55,7 @@ def check_golden_file(
     # if updating a golden file that doesn't exist yet, load the output again
     if update_golden and not golden_exists:
         golden = load_json_file(output_path)
+        shutil.copyfile(str(output_path), str(golden_path))
```

Comment:
If the golden doesn't exist, doesn't it make sense to just set it to None or `[]` -> there is a diff -> we update the underlying file

This logic seems a bit too complex, but we can address this in a follow up